sps20/content/venue/contents.lr

```diff
@@ -1,38 +1,51 @@
-title: Venue
+title: Location
```

Comment:
```suggestion
title: Venue
```

Whats wrong with venue?

Reply:
I lot of other conference side use location and i personally like location more. but i'm fine with both. On going discussion: https://swiss-python-summit.slack.com/archives/CB5962QQJ/p1718131716656189?thread_ts=1718053207.563329&cid=CB5962QQJ